server/schema/processed_file.go

```diff
@@ -32,6 +32,14 @@ func (pf *ProcessedFile) GetPayloadBytes(marshaller Marshaller) []byte {
 	return b
 }
 
+//GetPayloadUsingStronglyTypedMarshaller returns bytes, containing marshalled payload
+//StronglyTypedMarshaller needs to know payload schema (types of fields) to convert payload to byte slice
+func (pf *ProcessedFile) GetPayloadUsingStronglyTypedMarshaller(stm StronglyTypedMarshaller) []byte {
+	b, _ := stm.Marshal(pf.BatchHeader, pf.payload)
+	//TODO do i need error handling here?
```

Comment:
let's return an error if occurred. Handle this error in storages

Reply:
done